src/encord_active/analysis/metrics/sequence/missing_objects_and_wrong_tracks.py

```diff
@@ -9,6 +9,13 @@
 )
 from encord_active.analysis.types import AnnotationMetadata
 from encord_active.db.metrics import MetricType
+from encord_active.lib.common.utils import get_polygon, mask_iou
+
+
+def _filter_object(obj: Dict[str, MetricDependencies]) -> Dict[str, Polygon]:
```

Comment:
This function can be deleted - its not used

Reply:
nice catch!